src/backend/templates/partials/footer.hbs

```diff
@@ -28,11 +28,16 @@
          {{#if copyright}}
          {{#if copyright.licence}}
          <p>
-         <a href="{{{copyright.licence_url}}}"> <img src="{{{copyright.logo}}}"> </a>
-         &copy; {{copyright.year}}
+        <a href="{{{copyright.licence_url}}}"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
```

Comment:
>  Please use the small Creative Commons buttons instead of large buttons.

If I am not wrong, here @mariobehling asked to use the compact icon for creative common, that's why I hard coded it. Correct me if I understood it wrong. :sweat_smile:

I have given sample event in above comments. Both before and after changes.